sycl/include/CL/sycl/handler.hpp

```diff
@@ -62,7 +62,7 @@ class __copyAcc2Acc;
 namespace cl {
 namespace sycl {
 
-namespace csd = cl::sycl::detail;
+namespace csd = ::cl::sycl::detail;
```

Comment:
I believe we should remove this alias at all because we shouldn't define any internal things in just cl::sycl .